### ISSUE TITLE
How to use these tokens with DS ?

### ISSUE THREAD

**Issue by dmytro-infineon** [Infineon/Infineon-Design-System-Tokens#33]
Hello @tishoyanchev @verena-ifx 

Should I install this package alongside `https://github.com/Infineon/infineon-design-system-stencil` ?
For example I am importing `<IfxNavbar />` component and Application's name does not have fonts connected. I have browser's fonts only.

Is there a way to apply Design system's `:root` styles in my app (React+TS), so to say ? 

I've seen that this package is in `infineon-design-system-stencil/packages/components-react/package.json` dependencies. So I was wandering if I have to use it as well.

Basically I just bootstrapped a new app and I'd love to have these basic styles to be applied.

I did 
```
import { defineCustomElements } from '@infineon/infineon-design-system-react';
defineCustomElements(window);
```
in my `main.tsx`

Could you please help me here? :)

**Comment by tishoyanchev** [comment]
Hi! @dmytro-infineon 
No, you don't need to install this package. It's already included as a dependency in our main component's package.

There was an issue with the font family, and that's why the Navbar's application-name had a different font-family. We've been able to resolve the issue, and you should be seeing the correct font-family from now on. Do let us know if there's still a problem.

General note: If you only want to use our tokens without necessarily using the code components, there are two ways to do it: 
1) By installing our npm package, which will make all the tokens available to you, and;
2) By using the CDN link, which you can find in the README of our repository. 

Also, we have a webex group chat for technical and or general questions. You can find the link to it in our brandportal (goto/dds).

Let us know if you still have any questions!

**Comment by dmytro-infineon** [comment]
Hey @tishoyanchev 
Now I'm on version ```"@infineon/infineon-design-system-react": "^20.43.8"``` but still have font issue which looks like (in Firefox):
<img width="368" alt="Screenshot 2024-01-23 at 19 27 36" src="https://github.com/Infineon/Infineon-Design-System-Tokens/assets/151577238/e700d8e2-817d-4764-ae70-af8054a50b43">

In webkit browsers I simply have ```font-family: -webkit-standard;``` in computed properties.

As of now I also don't have Webex access, working on that, so for me the only way to communicate with you is in GH issues ;) 

So, my `App.tsx` approximately looks like this:
```
import './App.css';
import { IfxNavbar, IfxNavbarMenu, IfxNavbarMenuItem } from '@infineon/infineon-design-system-react';

function App() {
  return (
    <>
      <IfxNavbar application-name="Infineon Platform">
        <IfxNavbarMenu>
          <IfxNavbarMenuItem href="">Module 1</IfxNavbarMenuItem>
          <IfxNavbarMenuItem href="">Module 2</IfxNavbarMenuItem>
          <IfxNavbarMenuItem href="">Module 3</IfxNavbarMenuItem>
        </IfxNavbarMenu>
      </IfxNavbar>

     ...

      <OtherComponent />  
    </>
  );
}

export default App;
```  
and my `Navbar` looks like:

<img width="1105" alt="Screenshot 2024-01-23 at 19 43 19" src="https://github.com/Infineon/Infineon-Design-System-Tokens/assets/151577238/151f3c10-2e3d-4ffb-8f9e-8d2f3c51db56">

Could you please maybe suggest what could go wrong?

**Comment by tishoyanchev** [comment]
@dmytro-infineon 
Hi,
Silly question, but have you deleted your default React styles defined in App.css and Index.css? 
If you are sure that you don't have a custom font-family defined somewhere in your App, which would override our font-family, then try deleting the cache. Also try on another browser. You could also try re-installing the package.
I can't reproduce the issue, but if it keeps happening for you, try setting the font-family manually: "source sans 3"
